dev/integration_tests/channels/ios/Runner/AppDelegate.m

```diff
@@ -86,38 +86,36 @@ @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
   [GeneratedPluginRegistrant registerWithRegistry:self];
   // Override point for customization after application launch.
-  FlutterViewController *flutterController =
-      (FlutterViewController *)self.window.rootViewController;
-
+  id<FlutterPluginRegistrar> registrar = [self registrarForPlugin:@"platform-channel-test"];
```

Comment:
is setting up channels in didFinishLaunching still recommended approach after our latest discussion?